test/test_parser_multipart.rb

```diff
@@ -30,7 +30,7 @@ def test_parse_values_with_colon_data
   end
 
   def test_parse_with_error
-    raw = "\r\n--89467f8e0c6b48158c8f1883910212ec\r\nContent-Type: text/xml\r\nContent-ID: foo\r\nObject-ID: *\r\n\r\n<RETS ReplyCode=\"20403\" ReplyText=\"No Object Found\" />\r\n\r\n--89467f8e0c6b48158c8f1883910212ec--\r\n"
+    raw = "\r\n--89467f8e0c6b48158c8f1883910212ec\r\nContent-Type: text/xml\r\nContent-ID: foo\r\nObject-ID: *\r\n\r\n<RETS ReplyCode=\"999\" ReplyText=\"An Unexplaned Error\" />\r\n\r\n--89467f8e0c6b48158c8f1883910212ec--\r\n"
```

Comment:
This test is actually testing that the MultiParser successfully calls out to `Rets::Client::ErrorChecker`. This is the only error we test here out of:

```
NoRecordsFound
NoObjectFound
InvalidRequest
HttpError
AuthorizationFailure
```

Which can all be raised in the ErrorChecker and thus come out here.